optim.py

```diff
@@ -143,9 +143,9 @@ def optimize(model,
         val_loader    = [DataLoader] validation data loader
         val_writer    = [SummaryWriter] TensorBoard writer of validation metrics
```

Comment:
Type should be `MetricWriter` for `val_writer` I believe as all the other writers are of this type.